### PR TITLE
Refactoring code to allow Translations

### PR DESCRIPTION
**Pull request by chrosey**
started by adding all Strings visible to Users into Resources and adding Translations to German. 

Refactored setPokemonMarkers to allow Pokemon names based on Language.

Refactored Toasting... saving some lines with MainActivity.toastMe()

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
-org.gradle.jvmargs=-Xmx1536m
+# Default value: -Xmx1024m -XX:MaxPermSize=256m
+# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+#
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+#Mon Jul 25 11:43:33 CEST 2016
+systemProp.http.proxyHost=10.19.241.33
```
Comment (Kolyall) [inline gradle.properties:24]: seems it's not necessary

File: app/src/main/java/com/omkarmoghe/pokemap/views/MainActivity.java [hunk 2]
```diff
@@ -37,6 +37,12 @@
 
     public static Toast toast;
 
+    public static void toastMe(String message){
```
Comment (Kolyall) [inline app/src/main/java/com/omkarmoghe/pokemap/views/MainActivity.java:53]: the method can be moved to BaseActivity

**Comment by omkarmoghe** [comment]
check the conflicts

File: gradle.properties (hunk 1 above)
Comment (chrosey) [inline gradle.properties:24]: oh... forgot to drop that. its was needed at first setup for my proxy

**Comment by nekoboy93** [comment]
i want help with spanish translation

File: app/src/main/java/com/omkarmoghe/pokemap/views/BaseActivity.java [hunk 3]
```diff
@@ -24,18 +25,18 @@ protected void onCreate(Bundle savedInstanceState) {
     }
```
Comment (rohanjain1) [inline app/src/main/java/com/omkarmoghe/pokemap/views/BaseActivity.java:26]: Please avoid whitespace commits

**Comment by fess89** [comment]
good one. as this one gets merged, I can help with Russian and Ukrainian translations

**Comment by s7092910** [comment]
Also can you fix all the <<<<<<<Head or any of the <<<< lines.

**Comment by chrosey** [comment]
where do these come from. i resolved conflicts and everywhere these lines where added.... i did mess it up somehow.

**Comment by s7092910** [comment]
They come from merge conflicts where you haven't exactly resolved the conflicts. And it would just combine the two conflicts into one with those lines indicating where they came from.

**Comment by silverscorp** [comment]
I help with the Brazilian/Portuguese translation.

**Comment by chrosey** [comment]
Closing this. Messed up my branch...
opened another PR  [here](https://github.com/omkarmoghe/Pokemap/pull/174) with working solution

**Pull request closed without merging**